app-shells/zsh/zsh-5.3.1.recipe

```diff
@@ -3,13 +3,13 @@ DESCRIPTION="Zsh is a shell designed for interactive use, although it \
 is also a powerful scripting language. Many of the useful features of \
 bash, ksh, and tcsh were incorporated into zsh; many original features \
 were added."
-HOMEPAGE="http://zsh.sourceforge.net"
-COPYRIGHT="1992-2014 Paul Falstad, Richard Coleman, Zoltán Hidvégi, \
+HOMEPAGE="https://zsh.org/"
+COPYRIGHT="1992-2017 Paul Falstad, Richard Coleman, Zoltán Hidvégi, \
 Andrew Main, Peter Stephenson, Sven Wishnowsky, and others"
 LICENSE="ZSH"
-REVISION="2"
+REVISION="1"
 SOURCE_URI="http://sourceforge.net/projects/zsh/files/zsh/$portVersion/zsh-$portVersion.tar.xz"
-CHECKSUM_SHA256="74e9453b5470b3c0970f9f93cfd603d241c3d7b1968adc0e4b3951073e8d3dec"
+CHECKSUM_SHA256="fc886cb2ade032d006da8322c09a7e92b2309177811428b121192d44832920da"
 PATCHES="zsh-$portVersion.patchset"
 
 ARCHITECTURES="x86_gcc2 x86 x86_64"
```

Comment:
You are using $secondaryArchSuffix and
```
ARCHITECTURES="x86_gcc2 x86 x86_64"
SECONDARY_ARCHITECTURES="x86"
```
but the recipe provides no libs, so you should either:
* remove support for secondary architectures,
* **_or_** mark x86_gcc2 as broken (if that's the case).